### ISSUE TITLE
Owlbot leaves pull requests conflict status on README.md

### ISSUE THREAD

**Issue by suztomo** [googleapis/repo-automation-bots#3293]
In the Java repositories, we sometimes observe that Owlbot makes pull request in conflict status and it's not mergable.

<img width="953" alt="Screen Shot 2022-03-14 at 6 00 59 PM" src="https://user-images.githubusercontent.com/28604/158298735-b7f48f6d-21c2-4732-a04c-2cbf629c3026.png">

From Neenu, I heard that It's often the case that newer version of Owlbot pull request has been merged to main branch and it causes the pull request in conflict status. It's always README.md. Currently we manually resolve the conflict when it happens and we know we need the pull request.

CC: @Neenu1995 @chingor13

**Comment by suztomo** [comment]
https://github.com/googleapis/java-shared-dependencies/pull/624 This hit conflict on `.github/.OwlBot.lock.yaml` and `.kokoro/build.bat`. Neenu knew these "chore: fix license header in build.bat" can be closed. If Owlbot knows this can be cloased, then Owlbot should close this.

**Comment by SurferJeffAtGoogle** [comment]
This happens only when

1. someone makes change to Java's templates in googleapis/synthtool
2. Owl Bot creates pull request with corresponding changes
3. nobody merges the pull request
4. someone makes another change to Java's templates in googleapis/synthtool
5. Owl Bot creates another pull request with corresponding changes
6. someone merges the pull request from 2 or 5.  
7. now there's a merge conflict in the remaining pull request

There's a potential fix that can make this less frequent, but not eliminate it 100%.

To avoid this situation, merge pull requests from Owl Bot as quickly as possible.

When this situation is unavoidable, you may want to consider the version history of the post processor docker image by running 

```
$ gcloud beta container images list-tags gcr.io/cloud-devrel-public-resources/owlbot-java
DIGEST        TAGS            TIMESTAMP
b0b1c1c89570  1155a97,latest  2022-03-07T14:12:53
df8d7b2cc0db  4fea5f4         2022-03-02T10:35:29
e3746f849555  1622741         2022-03-01T13:44:41
387835a1375a  e122cb0         2022-02-25T11:00:42
f40929ecc45d  3b0a0e2         2022-02-14T14:56:10
3c950ed12391  53a58c2         2022-02-07T15:35:40
ebc2104854c5  5b77727         2022-02-07T12:18:12
7c853edc4136  010e6b0         2022-02-03T10:22:55
7062473f423f  466412a         2022-01-25T14:49:26
56de8d416e8e  0226158         2022-01-19T10:21:18
```

**Comment by SurferJeffAtGoogle** [comment]
With the work-arounds described in this issue, I'm demoting it to P3.

**Comment by suztomo** [comment]
Yes, the number of conflicts are now small. Manual work around is good.

**Comment by suztomo** [comment]
Thank you!
